app/code/Codifi/Training/view/frontend/web/js/popup.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * Codifi_CustomerRequest
+ *
+ * @copyright   Copyright (c) 2021 Codifi
+ * @author      Pavel Zelenevich <pzelenevich@codifi.me>
+ */
+
+require([
+    'jquery',
+    'Magento_Customer/js/customer-data',
+    'Magento_Ui/js/modal/confirm',
+], function ($, customerData) {
+    'use strict';
+
+    customerData.reload('credit_hold_section');
```

Comment:
See example how to work with component
vendor/magento/module-checkout/view/frontend/templates/registration.phtml
vendor/magento/module-checkout/view/frontend/web/js/view/registration.js